apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -141,8 +140,74 @@ const WriteInActionButton = styled(Button)`
 
 const SectionLabel = styled.div`
   font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
-  margin-top: 1rem;
   margin-bottom: 0.5rem;
+
+  :not(:first-child) {
+    margin-top: 1rem;
+  }
+`;
+
+// styles closely imitate our RadioGroup buttons, but we don't use RadioGroup
+// because we need to be able to deselect options by clicking them again
+const CandidateStyledButton = styled(Button)`
+  border-color: ${(p) => p.theme.colors.outline};
+  border-width: ${(p) =>
+    p.theme.sizeMode === 'desktop'
+      ? p.theme.sizes.bordersRem.thin
+      : p.theme.sizes.bordersRem.hairline}rem;
+  flex-wrap: nowrap;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.regular};
+  justify-content: start;
+  padding-left: 0.5rem;
+  text-align: left;
+  width: 100%;
+
+  /* Increase contrast between selected/unselected options when disabled by
+   * removing the darkening filter for unselected options. */
+  &[disabled] {
+    ${(p) => p.color === 'neutral' && `filter: none;`}
+  }
+`;
+
+function CandidateButton({
+  candidate,
+  isSelected,
+  isSelectedStatusUpToDate,
+  onSelect,
+  onDeselect,
```

Comment:
I might consolidate these to one `onChange` prop and let the caller worry about `isSelectedStatusUpToDate`